modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/commerce-catalogs/test-commerce-catalog-1.products.specifications.json

```diff
@@ -0,0 +1,13 @@
+[
+	{
+		"categoryKey": "test-commerce-specification-key-1",
+		"cpDefinitionExternalReferenceCode": "TESTCOMMERCEPRODUCT1",
+		"description": "",
+		"facetable": false,
+		"key": "test-product-specification-1",
+		"productSpecificationValue": {
+			"en_US": "1"
+		},
+		"title": "Test Product Specification 1 update"
+	}
+]
```

Comment:
lets add more specifications
like catalog we can put 2 specification in bundle-1 and 3 in bundle-2